test/specs/fetch.specs.ts

```diff
@@ -0,0 +1,63 @@
+import { expect } from "chai";
+import JSONBig from "json-bigint";
+
+import { APIError } from "../../src/errors/APIError";
+import { checkStatus, parseJSON } from "../../src/utils/fetch";
+
+const createResponse = (data: Record<string, unknown>, status = 200) =>
+    new Response(JSONBig({ useNativeBigInt: true }).stringify(data, null, 2), {
+        "status": status,
+        "statusText": "Dummy status",
+    });
+
+describe("Fetch Helpers", () => {
+    describe("parseJSON", () => {
+        it("return the parsed response body", async () => {
```

Comment:
can you clarify what this is testing?

Reply:
This is testing that the response body get parsed, which is what the function does. This is the general case to see if everything goes as expected when provided a response.

Reply:
if you're asserting `parseJson` result, perhaps a unit test for `parseJson` would be sufficient? 🤔 and in this case here we could probably assert the stringified json instead 🤔

Reply:
Note that is is testing with a response. We do get a response in parameter and parse its body. The JSON.parse does not get the response body here. Some other things are happening in here as well. As this is a generic utils it is better to have basic tests here.

Maybe I do not understand what you mean though 🤔

Reply:
oohh... i see, my bad! 🙇